platform/admin/ui/src/test/resources/ApplicationTree.json

```diff
@@ -1 +1 @@
ic fields to query the Catalog and other Sources. The right pane consists of a map.\\nIt includes both standard (3d globe) and simple (text page) versions.::Standard Search UI","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.ui.search\/search-ui-app\/2.8.0-SNAPSHOT\/xml\/features"},{"children":[],"name":"solr-app","description":"The Solr Catalog Provider (SCP) is an implementation of the CatalogProvider interface using Apache Solr 4.7.2 as a data store.\\nThe SCP supports extensible metacards, fast\/simple contextual searching, indexes xml attributes\/CDATA sections\/XML text elements, contains full XPath support, works with an embedded local Solr Server, and also works with a standalone Solr Server.::Solr Catalog","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.catalog.solr\/solr-app\/2.8.0-SNAPSHOT\/xml\/features"}],"name":"catalog-app","description":"The Catalog provides a framework for storing, searching, processing, and transforming information.\\nClients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.\\nAt the core of the Catalog functionality is the Catalog Framework, which routes all requests and responses through the system, invoking additional processing per the system configuration.::Catalog","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.catalog\/catalog-app\/2.8.0-SNAPSHOT\/xml\/features"},{"children":[{"children":[],"name":"admin-app","description":"Administration application for installation and management.\\nIncludes the Admin UI and the underlying application service that supports the user interface.\\nThe Admin UI provides contains modules allowing the administrator to install\/remove applications and their dependencies and to access configuration pages to customize and tailor system services and properties.\\nThe application service provides the supporting operations allowing the Admin UI to add, remove, start, stop, and obtain status information about all applications on the system.::Admin","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.admin\/admin-app\/2.8.0-SNAPSHOT\/xml\/features"}],"name":"security-services-app","description":"The Security Application provides Authentication, Authorization, and Auditing services.\\nThey comprise both a framework that developers and integrators can extend and also a reference implementation that can be used which meets security requirements.::Security","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.security\/security-services-app\/2.8.0-SNAPSHOT\/xml\/features"}],"name":"platform-app","description":"Installs the platform boot features which all other applications depend upon.\\nPlatform features installed by default include Apache CXF, Apache Camel, Action Framework, MIME Framework, Metrics, Security Core API and Security Encryption","state":"ACTIVE","version":"2.8.0-SNAPSHOT","uri":"mvn:ddf.platform\/platform-app\/2.8.0-SNAPSHOT\/xml\/features"}],"timestamp":1441833007,"status":200}
```

Comment:
✏️  We may want to fix the sentence that says `The Admin UI provides contains` and remove one of the two verbs.